formcentral/.gitignore

```diff
@@ -1 +1 @@
-.env*
```

Comment:
Are we using `.env*`? If so, could you point me to where it is being used?

Reply:
I just use it to pass the env vars for development, but if you create a development container, we no longer need them!